editoast/Cargo.toml

```diff
@@ -14,15 +14,17 @@ clap = { version = "~4.0.26", features = ["derive", "env"] }
 colored = "2.0.0"
 chrono = { version = "0.4.23", features = ["serde"] }
 derivative = "2.2.0"
-diesel = { version = "1.4.8", features = ["postgres", "uuidv07", "serde_json", "chrono"] }
+diesel = { version = "2.0", features = ["postgres", "serde_json", "chrono"] }
 json-patch = "0.3.0"
 rand = "0.8.5"
 reqwest = { version = "~0.11.13", features = ["json"] }
-rocket = { version = "0.5.0-rc", features = ["json"] }
-rocket_sync_db_pools = { version = "0.1.0-rc", features = [
+rocket = { git = "https://github.com/SergioBenitez/Rocket.git", rev = "162fafa0", features = [
+    "json",
+] }
+rocket_sync_db_pools = { git = "https://github.com/SergioBenitez/Rocket.git", rev = "162fafa0", features = [
     "diesel_postgres_pool",
 ] }
-rocket_cors = { git = "https://github.com/lawliet89/rocket_cors.git", rev = "54fae070" }
+rocket_cors = { git = "https://github.com/lawliet89/rocket_cors.git", rev = "c17e8145" }
```

Comment:
The CI doesn't pass on this commit. https://github.com/lawliet89/rocket_cors/actions/runs/2990544894.

Reply:
I reverted it anyway since that commit doesn't really adds anything but I agree that the state of rocket isn't ideal...